examples/gaussian_process/plot_gpr_dim_selection.py

```diff
@@ -0,0 +1,81 @@
+"""
+=========================================================
+Illustration of SelectDimensionKernel
+=========================================================
+A simple two-dimensional regression example computed in using two
```

Comment:
You need an empty line between line 4 and 5, otherwise the documentation does not build.